app/assets/javascripts/application.js

```diff
@@ -1,5 +1,10 @@
 //= require govuk_publishing_components/dependencies
-//= require govuk_publishing_components/all_components
+//= require govuk_publishing_components/components/checkboxes
```

Comment:
according to [a conversion on Transition's gem upgrade pull request](https://github.com/alphagov/transition/pull/1647#discussion_r1681137070), `checkboxes` needs moving into the ES6-specific file, too

Reply:
OK, that's sorted now